packages/utilities/shared-utils/src/functions.ts

```
@@ -79,3 +79,62 @@ export function objectToDeps(obj: Extractable) {
+export const kebabCase = (s: string) => {
+  return s.replace(/([a-z])([A-Z])/g, "$1-$2").toLowerCase();
+};
+
+export const mapKeys = (
+  obj: Record<string, any>,
+  iteratee: (value: any, key: string) => any,
+): Record<string, any> => {
+  const res: Record<string, any> = {};
+
+  for (const key in obj) {
+    if (obj.hasOwnProperty(key)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)